kong/db/strategies/off/init.lua

```diff
@@ -173,7 +168,10 @@ local function page_for_key(self, key, size, offset, options)
     -- The rest of entities' lists (i.e. "services|<ws_id>|@list") only contain ids, so in order to
     -- get the entities we must do an additional cache access per entry
     else
-      item = cache:get(item, nil, nil_cb)
+      item, err = unmarshall(lmdb_get(item))
+      if err then
```

Comment:
i've noticed some are of the form
```lua
item, err = unmarshall(lmdb_get(item))
if err then ...
```
while others (in this same file) are
```lua
item, err = unmarshall(lmdb_get(item))
if not item then ....
```
any reason for the difference?  (i usually favor the `if not item then...` style, but sometimes it can't be done)